x/pki/x509/x509.go

```diff
@@ -67,16 +70,14 @@ func PatchCertificate(certificate Certificate) Certificate {
 	newVIDKey := "vid"
 	newPIDKey := "pid"
 
-	issuer := certificate.Issuer
-	issuer = FormatOID(issuer, oldVIDKey, newVIDKey)
-	issuer = FormatOID(issuer, oldPIDKey, newPIDKey)
-
 	subject := certificate.Subject
 	subject = FormatOID(subject, oldVIDKey, newVIDKey)
 	subject = FormatOID(subject, oldPIDKey, newPIDKey)
 
-	certificate.Issuer = issuer
-	certificate.Subject = subject
+	certificate.SubjectAsText = subject
+
+	certificate.Issuer = StringtoBase64(certificate.Issuer)
```

Comment:
`Issuer` and `Subject` fields can be set when initializing the certificate like
```
	certificate := Certificate{
		Issuer:         StringToBase64(cert.Issuer.String()),
		SerialNumber:   cert.SerialNumber.String(),
		Subject:        StringToBase64(cert.Subject.String()),
		SubjectAsText:  cert.Subject.String(),
		SubjectKeyID:   BytesToHex(cert.SubjectKeyId),
		AuthorityKeyID: BytesToHex(cert.AuthorityKeyId),
		Certificate:    cert,
	}
```

Reply:
Done! ✅